src/containers/Registration/Registration.tsx

```diff
@@ -55,11 +60,15 @@ const Registration: React.FC = () => {
   const { login } = useAuth();
   const history = useHistory();
 
+  const checkFromValidation = () => {
```

Comment:
We can wrap this shit into `useMemo` so it becomes memoized value instead of a function. Something like:
```
const isValid = useMemo(() => { /* the same code */ }, [ /* dependencies */ ]);
```